From 74a393e0a8cab3d1b6ae8f7a5d6869a107455a04 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 6 Oct 2019 15:31:14 +0200
Subject: [PATCH 1/1] configs: sheevaplug disable EFI_UNICODE_CAPITALIZATION

With upcoming patches u-boot.kwb gets too large.

Save 3740 bytes in u-boot.kwb by disabling Unicode support in the UEFI
sub-system.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 configs/sheevaplug_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/sheevaplug_defconfig b/configs/sheevaplug_defconfig
index 734e894c98..cc48536415 100644
--- a/configs/sheevaplug_defconfig
+++ b/configs/sheevaplug_defconfig
@@ -46,3 +46,4 @@ CONFIG_DM_USB=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_STORAGE=y
 CONFIG_LZMA=y
+# CONFIG_EFI_UNICODE_CAPITALIZATION is not set
-- 
2.23.0

